units/heroes/Running_Heroes.cfg

```diff
@@ -0,0 +1,203 @@
+#textdomain wesnoth-Danse_Macabre
+[unit_type]
+    id=Running Hero
+    [base_unit]
+        id=Running Corpse
+    [/base_unit]
+    advances_to=Brainless Hero
+
+    # should be enabled in the end, since we
+    # a) don't want players in the beginning to see what heros will appear later
+    # b) nor do they have to see how the heroes look on higher levels (keeping some surprise)
+    # c) like the base_unit, they are shown in the help as advancements of the Soulless, which looks a bit odd
```

Comment:
This can be solved with this code:
> 
    [advancefrom]
        unit=Soulless Hero
        experience=35
    [/advancefrom]